aspnetcore/publishing/linuxproduction.md

```diff
@@ -160,7 +181,7 @@ Main PID: 9021 (dotnet)
             └─9021 /usr/local/bin/dotnet /var/aspnetcore/hellomvc/hellomvc.dll
 ```
 
-With the reverse proxy configured and Kestrel managed through systemd, the web application is fully configured and can be accessed from a browser on the local machine at `http://localhost`. Inspecting the response headers, the **Server** still shows the ASP.NET Core application being served by Kestrel.
+With the reverse proxy configured and Kestrel managed through systemd, the web application is fully configured and can be accessed from a browser on the local machine at `http://localhost`. Inspecting the response headers, the `Server` header shows the ASP.NET Core application being served by Kestrel.
```

Comment:
Might be worth pointing out that it should also be accessible from a remote machine barring any firewall that might be in the way.